### PR TITLE
[fix bug 1181162] Whatsnew Firefox 40 development

### PR DESCRIPTION
**Pull request by alexgibson**

### CODE REVIEW

File: bedrock/firefox/templates/firefox/whatsnew_40/index.html [hunk 1]
```diff
@@ -0,0 +1,99 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/base-resp.html" %}
+
```
Comment (flodolo) [inline bedrock/firefox/templates/firefox/whatsnew_40/index.html]: Let's use ```/firefox/whatsnew_40.lang``` for this

```
{% add_lang_files "firefox/whatsnew_40" %}
```

File: bedrock/settings/static_media.py [hunk 2]
```diff
@@ -541,6 +541,13 @@
         ),
         'output_filename': 'css/firefox_whatsnew_38_pocket-bundle.css',
     },
+    'firefox_whatsnew_40': {
+        'source_filenames': (
+            'css/tabzilla/tabzilla-static.less',
+            'css/firefox/whatsnew_40/whatsnew-40.less',
+        ),
+        'output_filename': 'css/firefox_whatsnew_38_pocket-bundle.css',
```
Comment (flodolo) [inline bedrock/settings/static_media.py]: Is this (and same below) correct?

Comment (alexgibson) [inline bedrock/settings/static_media.py]: Doh, thanks!

File: bedrock/firefox/templates/firefox/whatsnew_40/index.html (hunk 1 above)
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/whatsnew_40/index.html]: Will do :+1:

**Comment by alexgibson** [comment]
Closing this PR - page is being re-purposed for another URL instead.

**Pull request closed without merging**